apps/authentication/backends/sso.py

```diff
@@ -61,23 +45,15 @@ def is_enabled():
         return settings.AUTH_LARK
 
 
-class SlackAuthentication(JMSModelBackend):
-    """
-    什么也不做呀😺
-    """
-
+class SlackAuthentication(JMSBaseAuthBackend):
     @staticmethod
     def is_enabled():
         return settings.AUTH_SLACK
 
-    def authenticate(self, request, **kwargs):
+    def authenticate(self):
         pass
 
 
-class AuthorizationTokenAuthentication(JMSModelBackend):
-    """
-    什么也不做呀😺
-    """
-
-    def authenticate(self, request, **kwargs):
+class AuthorizationTokenAuthentication(JMSBaseAuthBackend):
+    def authenticate(self):
         pass
```

Comment:
There don't seem to be any major inconsistencies or issues with the provided JMS authentication backend classes. The key differences between them appear to be class names (such as `SSOAuthentication` or `SlackAuthentication`) and whether they implement custom methods (`authenticate`), but these differences shouldn’t impact performance or functionality.

If you need further detailed comments, let's say more about each specific issue that needs clarification or if certain parts of your question isn't clear enough:

1) Is there something in particular you're looking for clarifications on?
2) Have you noticed anything unexpected or unusual behavior related to the changes? Or are you interested in general understanding of this project structure?

I'll provide tailored responses after considering those points so I can assist you effectively!